src/pages/signup/Signup.js

```diff
@@ -1,7 +1,9 @@
 import { useState } from "react";
 import { Link, useNavigate } from "react-router-dom";
 import Navbar from "../../components/navbar/Navbar";
+import LogedNavbar from "../../components/navbar/LogedNavbar";
 import { useAuth } from "../../context/AuthContext";
+import { auth } from "../../utils/firebase";
```

Comment:
components should not import directly from firebase; instead "user" should be exported from the useAuth context.
This will also allow to show the user when hard-refreshing after logging in.